tests/cases/fourslash/todoComments20.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path='fourslash.ts' />
+
+// @Filename: dir1/node_modules/todoTest0.ts
+//// // TODO

```

Comment:
I don't understand these last 2 tests. They seem to be verify the comment IS present, when based on the file names shouldn't they be NOT present? Or am I reading the tests wrong?